README.rst

```diff
@@ -62,7 +69,7 @@ For Gitlab CI
      - docker:dind
    before_script:
      - apk add --update bash coreutils tar
-     - git clone https://github.com/ros-industrial/industrial_ci .industrial_ci
+     - git clone --quiet --depth 1 https://github.com/ros-industrial/industrial_ci .industrial_ci
```

Comment:
+1

Btw, what about bitbucket section ([line 94](https://github.com/ros-industrial/industrial_ci/pull/337/files#diff-88b99bb28683bd5b7e3a204826ead112R94))?

Reply:
I haven't scrolled that far down.. updated :)